app/services/scrapers/scraper_manager.py

```
@@ -13,12 +13,12 @@ class ScraperManager:
 
     bluesky_scraper: Optional[BlueskyScraper] = None
     weibo_scraper: Optional[WeiboScraper] = None
-    firecrawl_scraper: Optional[FirecrawlScraper] = None
+    general_scraper: Optional[GeneralScraper] = None
 
     scrapers = {"bluesky": bluesky_scraper,
                 "weibo": weibo_scraper,
-                "other": firecrawl_scraper,
-                "unknown": firecrawl_scraper}
+                "other": general_scraper,
+                "unknown": general_scraper}
```

Ruff (0.15.0)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)